frontend_app_file/src/container/PopupCreateFile.jsx

```diff
@@ -65,32 +105,28 @@ class PopupCreateFile extends React.Component {
     }
   })
 
-  handleChangeFile = newFile => {
-    if (!newFile || !newFile[0]) return
+  handleChangeFile = newFileList => {
+    const { props, state } = this
 
-    const fileToSave = newFile[0]
+    if (!newFileList || !newFileList[0]) return
```

Comment:
Maybe it's better to test the length of newFileList, but I'm not sure about what you exactly test here.
What I'm sure about is that if `newFileList.length === 0`, this line will crash.